docs/configuration/configuration.md

```diff
@@ -1360,7 +1360,7 @@ interface.
 The following meta labels are available on targets during [relabeling](#relabel_config):
 
 * `__meta_openstack_address_pool`: the pool of the private IP.
-* `__meta_openstack_instance_flavor`: the flavor ID of the OpenStack instance.
+* `__meta_openstack_instance_flavor`: the flavor name or flavor ID of the OpenStack instance, whichever is available.
```

Comment:
not sure how we make it more explicit that the flavor name is preferred over flavor id. 
```suggestion
* `__meta_openstack_instance_flavor`: the flavor name of the OpenStack instance, or the flavor ID if the flavor name isn't available.
```

Reply:
For me that's ok.

Changes committed ✅

Reply:
Actually there is no preference, API before 2.47 only returned the flavor ID and the API after 2.47 only return the flavor name, they're mutually exclusive.
But ok for the wording.